packages/core/src/forms/forms.stories.ts

```diff
@@ -1899,6 +1842,34 @@ export const checkoutForm = () => {
       </div>
 
       <cds-button>continue to checkout</cds-button>
-    </div>
+    </cds-form-group>
   `;
 };
+
+export const responsiveCheckoutForm = () => {
+  useEffect(() => {
+    const iframe = document.querySelector<HTMLIFrameElement>('#complex-form-demo');
+    iframe.onload = () => {
+      const resizeObserver = new ResizeObserver(
+        () => (iframe.style.height = `${iframe.contentWindow.document.body.scrollHeight}px`)
+      );
+      resizeObserver.observe(iframe);
+    };
+  });
```

Comment:
This is for the responsive demo, the iframe is resizable with `resize: horizontal` css prop. This code will set the iframe hight to whatever the content of the iframe is.